### DISCUSSION TITLE
**System.MissingMethodException:**

### DISCUSSION THREAD

**Discussion by janguar** [PrismLibrary/Prism#2337] [Xamarin.Forms]
Hi all. How i can resolve **System.MissingMethodException:** 'Method not found: bool Xamarin.Forms.Internals.ResourceLoader.get_IsEnabled()', 

HELP PLEASE !!!!!

**Comment by dansiegel** [comment]
need some version numbers... sounds like you're doing something where there is binary incompatibility.
